onnxruntime/core/providers/qnn/builder/onnx_ctx_model_helper.cc

```diff
@@ -160,7 +160,7 @@ bool IsContextCacheFileExists(const std::string& customer_context_cache_path,
   if (!customer_context_cache_path.empty()) {
     context_cache_path = ToPathString(customer_context_cache_path);
   } else if (!model_pathstring.empty()) {
-    context_cache_path = model_pathstring + ToPathString("_qnn_ctx.onnx");
+    context_cache_path = model_pathstring + ToPathString("_ctx.onnx");
```

Comment:
since this particular implementation is for qnn, i'm wondering if we should keep qnn in the name?

Reply:
Eventually, we will support multi-partition and move this model saving to inference session level. We do need to remove EP specific label.
I remove this qnn label and updated the inference code accordingly so that we don't need to change the inference sample code again once the multi-partition support get in.

---
In reply to: [1432352771](https://github.com/microsoft/onnxruntime/pull/18877#discussion_r1432352771) [](ancestors = 1432352771)

Reply:
Also, this is the default name in case users do not set it. User should set their own name normally.

---
In reply to: [1432369068](https://github.com/microsoft/onnxruntime/pull/18877#discussion_r1432369068) [](ancestors = 1432369068,1432352771)